### DISCUSSION TITLE
Step Function

### DISCUSSION THREAD

**Discussion by modem158** [kalkih/mini-graph-card#835] [Q&A]
Anyone know if there is the ability to create a step function graph for a binary sensor that only updates on a change of state?  

I turned smoothing off but I end up with a bunch of triangles.  It would be nice if there were a way to hold the last state until it changes.

**Comment by modem158** [comment]
I just messed with the update_interval but it seems to have no effect (or I'm using it wrong)

**Comment by ildar170975** [comment]
Try with a bigger "points_per_hour".
